src/styles/style.css

```diff
@@ -0,0 +1,32 @@
+@import url(./wallet.css);
```

Comment:
Check this css too.

Reply:
I will used wallet.css for future purpose